src/toil/server/wes/toil_backend.py

```diff
@@ -63,15 +63,17 @@
 logging.basicConfig(level=logging.INFO)
 
 class ToilWorkflow:
-    def __init__(self, base_scratch_dir: str, state_store_url: str, run_id: str):
+    def __init__(self, base_work_dir: str, state_store_url: str, run_id: str):
         """
         Class to represent a Toil workflow. This class is responsible for
         launching workflow runs via Celery and retrieving data generated from
         them.
 
-        :param base_scratch_dir: The directory where workflows keep their
+        :param base_work_dir: The directory where workflows keep their
                                  output/scratch directories under their run
-                                 IDs.
+                                 IDs. Includes the MessageBus which requires
+                                 a reliable location, thus should always be
+                                 the work_dir specified by the user.
```

Comment:
I don't think you did anything to the places where `ToilWorkflow()` is constructed to ensure this new constraint, and I don't really understand what you mean. Right now `base_scratch_dir` comes from the Toil *Server*'s `work_dir`, whicyh comes from https://github.com/DataBiosphere/toil/blob/88d4589592eba433d10b3c835ef2a0be5653336a/src/toil/server/app.py#L57-L59. But that's not really "specified by the user"; it's a fixed parameter of the WES server that we set at server startup and which doesn't. as fart as I know, get overridden by the user-supplied workflows. We probably want to rename the server parameter *away* from `work_dir`, because of the potential for confusion with an individual workflow's `work_dir` parameter, which is different.

And when it is passe din here as a `base_scratch_dir`, it is still one fixed directiry for all the workflows, so it doesn't really include the message bus log; it contains all the workflows' message bus logs, in subdirectories by `run_id`.